Jenkinsfile

```diff
@@ -69,17 +109,75 @@ pipeline{
 				}
 			}
 		}
-		// This stage archives all logs and database backups produced by Orthoinference. It also archives the eligible/inferred files produced by orthoinference.
-		stage('Archive logs and backups'){
+		// This stage generates the graph database using the graph-importer module, and replaces the current graph db with it.
+		stage('Post: Generate Graph Database'){
+			steps{
+				script{
+					sh "git clone https://github.com/reactome/graph-importer"
+					dir("graph-importer"){
+						sh "mvn clean compile assembly:single"
+						withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+							sh "java -jar target/GraphImporter-jar-with-dependencies.jar --name ${env.RELEASE_CURRENT} --user $user --password $pass --neo4j /tmp/graph.db"
+							sh "sudo service tomcat7 stop"
+							sh "sudo service neo4j stop"
+							// This static script adjusts permissions of the graph.db folder and moves it to /var/lib/neo4j/data/databases/.
+							sh "sudo bash /var/lib/jenkins/changeGraphDatabase.sh"
```

Comment:
Is this script a specific command that Jenkins has sudo access to execute in the sudoers file?  If so, the script should probably have permissions of "700" and be owned by the "Jenkins" user to ensure only the owner of the script (or a sudo user) can modify the script contents or execute it to prevent malcious command being injected into the script.

Reply:
Is `changeGraphDatabase.sh` in source control? I don't see it in this PR...

Reply:
@jweiser -- Yes thank you for reminding me, I intended to make it executable only by Jenkins

@SolomonShorser-OICR  It's not in source control. Its functionality will be used at a few different points in the Release process, and since it handles doing something sensitive (adjusting permissions and such), I thought it was appropriate to just keep it at /var/lib/jenkins.

Reply:
Sure, but if something happens to the server and you (or someone else who doesn't know very well what's going on here) need to redeploy all of this on a new EC2 instance, what is inside `changeGraphDatabase.sh`?

Reply:
These the contents:
```
## Jenkins user builds the graph db during Release, but Neo4j user is needed to replace graph.db folder at /var/lib/neo4j/data/databases/.
## This script changes all necessary permissions from Jenkins to Neo4j beforehand.
chmod 644 /tmp/graph.db/*
chmod a+x /tmp/graph.db/schema/
chown -R neo4j:adm /tmp/graph.db/
mv /var/lib/neo4j/data/databases/graph.db /var/lib/neo4j/data/databases/extra/
mv /tmp/graph.db /var/lib/neo4j/data/databases/
```

How do we handle this type of thing for pass2production? Would it maybe make sense to archive this in release-jenkins-utils or something?